### DISCUSSION TITLE
Add standard "tags" field to posts

### DISCUSSION THREAD

**Discussion by darian-parrish** [deso-protocol/dips#206] [Medium CIPs]
**Problem:** It is currently tough to filter posts based on the content it contains in the body. This means that apps are not able to easily build good content discovery interfaces.

**Current workarounds:** Hashtags in the body, Keyword analysis or computer vision(for images of course - discern photography vs. digital art for example).

**Use Cases:** Topic discovery, simple feed creation/filtering, hide NSFW content, community groups on chain(rather than Discord)

**Solution:** Add standard "tags" field to posts.

**Comment by sungkhum** [comment]
Yes, please!

**Reply by ChristoMeid** [reply]
This could also enable tags for content warnings, such as NSFW.
